### DISCUSSION TITLE
Independent hoist levels for tabs

### DISCUSSION THREAD

**Discussion by dipakc** [TriliumNext/Trilium#1561] [Ideas] [answered]
Is it possible to set different hoist levels for tabs? Currently, if I set change hoist note in one tab, it affects the view in the other tab. I would like multiple to have multiple tabs that that are hoisting different folders.

**Comment by zadam** [comment, marked as answer]
Hi, this is implemented in master, i.e. future 0.46 which will come out in a few weeks. In 0.45 this is not possible.

**Reply by dipakc** [reply]
Great! Thank you so much!
